controllers/comments.controller.js

```diff
@@ -21,4 +21,15 @@ exports.postComment =(req,res,next)=>{
     .catch((err)=>{
         next(err)
     })
+}
+
+exports.deleteComment=(req,res,next)=>{
+    const {comment_id} = req.params
+    removeComment(comment_id)
+    .then((comment)=>{
+        res.status(204).send({comment})
```

Comment:
For this one, you don't need to pass in the comment to the .send as we arn't looking to send back a body.

Your way is absolutely fine here but just so you're aware you can also use res.sendStatus(204) here because we arn't sending back a body.